packages/back-end/src/app.ts

```diff
@@ -145,6 +147,27 @@ if (ENVIRONMENT !== "production") {
   app.set("json spaces", 2);
 }
 
+if (stringToBoolean(process.env.PYTHON_SERVER_MODE)) {
+  // increase max payload json size to 50mb as a single query can return up to 3000 rows
+  // and we pass the results of all queries at once into python
+  app.use(
+    bodyParser.json({
+      limit: process.env.PYTHON_SERVER_INPUT_SIZE_LIMIT || "50mb",
```

Comment:
question: Should we increase the payload limit just to the `/stats` endpoint instead?

`app.post('/stats', bodyParser.json({...}), async (req, res) => {...})`;